pylint/checkers/classes.py

```diff
@@ -1418,11 +1427,12 @@ def _check_protected_attribute_access(self, node):
                     if _is_attribute_property(name, klass):
                         return
 
-                # A licit use of protected member is inside a special method
-                if not attrname.startswith(
-                    "__"
-                ) and self._is_called_inside_special_method(node):
-                    return
+                if not self.config.check_protected_access_in_special_methods:
```

Comment:
Maybe we can put the three conditions in one if ?

Reply:
Yes, good idea!